libcxx/test/std/utilities/meta/meta.unary/meta.unary.prop/common.h

```diff
@@ -39,9 +39,51 @@ struct A {
   A& operator=(const A&);
 };
 
-class Abstract
-{
-    virtual ~Abstract() = 0;
+class Abstract {
+  virtual ~Abstract() = 0;
 };
 
+// Types for reference_{constructs/converts}_from_temporary
+
+#if TEST_STD_VER >= 23
+
+struct NonPOD {
+  NonPOD(int);
+};
+enum Enum { EV };
+struct POD {
+  Enum e;
+  int i;
+  float f;
+  NonPOD* p;
+};
+// Not PODs
+struct Derives : POD {};
```

Comment:
These type names are confusing. Perhaps we only want to use `Base` and `Derived`.